packages/utilities/src/appendFunction.ts

```diff
@@ -0,0 +1,19 @@
+// tslint:disable:no-any
+
+/**
+ * Returns a single function which will call each of the given functions in the context of the
+ * parent.
+ */
+export function appendFunction<TRetVal>(parent: any, ...functions: (null | undefined | any)[]): () => TRetVal {
+  if (functions.length < 2) {
+    return functions[0] as () => TRetVal;
+  }
+
+  return (...args: any[]): TRetVal => {
+    let retVal: TRetVal | undefined = undefined;
+
+    functions.forEach((f: () => TRetVal) => f && (retVal = f.apply(parent, args)));
+
+    return (retVal as any) as TRetVal;
```

Comment:
Does this only return one return value, the last function called? Hmm, this is subtle. I wonder if this should be modified to support `void` functions only to ensure suitable application. i.e. `() => void` instead of `() => TRetVal`.

Reply:
yes subtle. I think i can comment on it, or i can return an array of retvals